plans.go

```diff
@@ -37,6 +38,33 @@ func (p Plan) String() string {
 	return Stringify(p)
 }
 
+// ReservationPricing - The reserved pricing for a plan
+type ReservationPricing struct {
+	OneYear   *OneYear   `json:"one_year"`
```

Comment:
`OneYear` and `ThreeYear` can share the same type, `Pricing` which already exists and fits the pattern.

Reply:
Will do.